splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/BaseFormView.jsx

```diff
@@ -376,7 +381,7 @@ class BaseFormView extends PureComponent {
     loadHook = (module, globalConfig) => {
         const myPromise = new Promise((myResolve) => {
             __non_webpack_require__([`app/${this.appName}/js/build/custom/${module}`], (Hook) => {
-                this.hook = new Hook(globalConfig, this.props.serviceName, this.state, this.util);
+                this.hook = new Hook(globalConfig, this.props.serviceName, this.state, this.props.mode, this.util);
```

Comment:
nit: Would it be more readable if we pass name as `form_mode`/`entity_mode`?

Reply:
okay, it's depends on Example Hook, Here only value(and position) is important, So We'll keep `form_mode/entity_mode` in example hook to make it more readable.

or we can change name of prop for the BaseFormView.js